models.py

```diff
@@ -1,4 +1,31 @@
+from datetime import datetime
+
 from room import db
+from flask_login import UserMixin
+from werkzeug.security import generate_password_hash, check_password_hash
+from room import login
+
+
+@login.user_loader
+def load_user(id):
+    return User.query.get(int(id))
+class ParadeMii(db.Model):
+    miiid = db.Column(db.Integer,primary_key=True)
+    logo1id = db.Column(db.String(5))
+    logobin = db.Column(db.String(8000)) # spotlightishere, can we make a server-side image encoder that maybe also resizes images?
+    # Thanks. -JG
```

Comment:
We definitely can! It has to be implemented before this is merged.

Reply:
Okay, I'll make a script for this and store it in `encodemii.py`, creative enough of a name :)

Reply:
I second that name 😄 
Also, I came up with a better name for room.py, how about runmii?